### ISSUE TITLE
ResultSet when using server jdbc is different than when using in-memory

### ISSUE THREAD

**Issue by tkolsen** [neo4j-contrib/neo4j-jdbc#39]
By testing i found that when getting data from server via jdbc, the resultset holds a json-string. This is not the case when using in-memory or embedded database. This causes problems when trying to test with in-memory database when using a server implementation in your program.

**Comment by jexp** [comment]
That's true, we noticed it ourself and have planned to unify that. We couldn't easily decide on a structure/type to support that caters both the embedded and remote case and so it got lost again.

But definitely rs.getString() should return the same representation.
Actually the remote rs.getObject() returns a map afaik.

**Comment by tkolsen** [comment]
Yes your right. The remote returns java.util.LinkedHashMap and the in-memory returns org.neo4j.kernel.impl.core.NodeProxy.
